src/neo4j/_async/io/_bolt5.py

```diff
@@ -665,3 +671,22 @@ def get_base_headers(self):
         headers = super().get_base_headers()
         headers["bolt_agent"] = BOLT_AGENT_DICT
         return headers
+
+
+class AsyncBolt5x4(AsyncBolt5x3):
+
+    PROTOCOL_VERSION = Version(5, 4)
+
+    def telemetry(self, api: TelemetryAPI, dehydration_hooks=None,
+                  hydration_hooks=None, **handlers) -> None:
+        if (
+            self.telemetry_disabled
+            or not self.configuration_hints.get("telemetry.enabled", False)
```

Comment:
Is self.configuration_hints configured/returned from the server?

Reply:
Yes. All configuration hints that come from the server are being stored on the connection object on `SUCCESS`ful `HELLO` https://github.com/neo4j/neo4j-python-driver/blob/fe85b713853edea0566c691aaebc6fd66c6ea9eb/src/neo4j/_async/io/_bolt5.py#L135

Reply:
Thanks for clarifying!